src/app/components/team/Statuses/StatusesComponent.js

```diff
@@ -93,7 +93,9 @@ const StatusesComponent = ({ team, setFlashMessage }) => {
 
   const handleDelete = ({ status_id, fallback_status_id }) => {
     const onCompleted = () => {
+      setStatuses(statuses.filter(s => s.id !== status_id));
```

Comment:
@danielevalverde I believe adding this resolves the delete not happening on the screen right away. Looking at the changes Darius made to the Language area, specifically `LanguageListItem.js line 133` in this PR, he is filtering out of the array the language that was deleted, so I just did the same, which seems to resolve that issue

Reply:
Yaaay!! I tested locally and its working for me :heart_eyes: :heart_eyes: